claudette/core.py

```diff
@@ -287,6 +315,8 @@ def __init__(self,
         assert model or cli
         assert cont_pr != "", "cont_pr may not be an empty string"
         self.c = (cli or Client(model))
+        if tools:
+            tools = [tool(t) for t in tools]
```

Comment:
```suggestion
        if tools: tools = [tool(t) for t in tools]
```

As a rule of thumb, aim for 1 idea<->1 line of code. That normally means you'll want single-line bodies of if/for on the same line, like this.